parser/prism/Translator.cc

```diff
@@ -996,8 +997,8 @@ unique_ptr<parser::Node> Translator::translate(pm_node_t *node) {
         }
         case PM_RESCUE_MODIFIER_NODE: {
             auto rescueModifierNode = reinterpret_cast<pm_rescue_modifier_node *>(node);
-            auto body = translate(reinterpret_cast<pm_node *>(rescueModifierNode->expression));
-            auto rescue = translate(reinterpret_cast<pm_node *>(rescueModifierNode->rescue_expression));
```

Comment:
These weren't necessary, since they were already `pm_node *`. This helper caught that! 🥳